styles/global.js

```diff
@@ -18,6 +18,12 @@ export default StyleSheet.create({
     fontFamily: 'ConcertOne-Regular',
     fontWeight: 'normal',
   },
+  boxShadow: {
+    shadowColor: 'black',
+    shadowOffset: { width: 4, height: 4 },
+    shadowOpacity: 0.1,
+    shadowRadius: 1
```

Comment:
Per the design in Figma, shouldn't the shadow radius be 10? It's a very hard edge shadow at 1, and in the design it's more diffuse.

Reply:
For now, I'll change it to 10.  

However, I'm going to take screenshots to compare what the page title component looks like when it has shadow radius of 1 vs. 10.  Then I'll let you and Britt compare and pick whichever you think looks better.